### ISSUE TITLE
On fullscreen bar always visible at bottom of the screen

### ISSUE THREAD

**Issue by Herkulisch** [mozilla-mobile/fenix#23691]
If a video enters fullscreen mode, one can still see a bar at the bottom of the screen. Also after exiting the fullscreen video, Firefox still hides the navigation bar.

**Comment by mcarare** [comment]
@Herkulisch Could you please specify the device on which the issue occurs, as well as the Android version? It would also be helpful if you tested this on more than one website and noted whether or not the issue reproduced on all of them.
Thank you!

**Comment by Herkulisch** [comment]
![Youtube in normal mode](https://user-images.githubusercontent.com/55994229/153637969-de097b4e-bf81-461a-ae31-5582de864c44.png)

![Youtube in private mode](https://user-images.githubusercontent.com/55994229/153639012-d0ee6fbe-1ef3-441a-b027-c0216452fb0b.png)

![This is on twitch.tv](https://user-images.githubusercontent.com/55994229/153639484-e64cc084-9ffd-4272-b3c0-414fc644195b.png)

I made a screenshot on youtube.com for example, but the issue also happens on other sites like twitch (third screenshot).

I am using a Google Pixel 6 running on Android 12 and if I have observed correctly this issue started with the latest update. It also happens in private mode, the toolbar is purple in that case. 

My installed Firefox Version right now is 97.1.0 (Build #2015860771)
